lib/openhab/core/dto/item_channel_link.rb

```diff
@@ -27,6 +27,11 @@ def item
         def channel_uid
           Things::ChannelUID.new(channelUID)
         end
+
+        # @return [String]
+        def to_s
```

Comment:
isn't this how it behaves already? (https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/link/AbstractLink.java)

Reply:
This is the DTO interface/class, not the same as the one in the link.
This DTO class is used in `ItemChannelLink[Added|Removed]Event`